extensions/2.0/Vendor/EXT_mesh_features/schema/featureId.schema.json

```diff
@@ -20,6 +20,11 @@
             "minimum": 0,
             "description": "A value that indicates that no feature is associated with this vertex or texel."
         },
+        "label": {
+            "type": "string",
+            "pattern": "^[a-zA-Z]\\w*$",
+            "description": "A label assigned to this feature ID set. Labels may contain only alphanumeric and underscore characters."
```

Comment:
Maybe mention that they must start with an alphabetic character? (and/or underscore depending on your answer to the previous question)